### ISSUE TITLE
import tensorflow error

### ISSUE THREAD

**Issue by fzzfbyx** [apple/tensorflow_macos#28]
I download and extract the release,create the new virtualenv and install to it.It occurs error when I import tensorflow in the virtualenv,which is:
    
from tensorflow.python._pywrap_tensorflow_internal import *
ImportError: dlopen(/Users/fzzfbyx/pythonvirtual/tff/lib/python3.8/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so, 6): no suitable image found.  Did find:
	/Users/fzzfbyx/pythonvirtual/tff/lib/python3.8/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so: mach-o, but wrong architecture
	/Users/fzzfbyx/pythonvirtual/tff/lib/python3.8/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so: mach-o, but wrong architecture
Failed to load the native TensorFlow runtime.

And I check my python
$ file $(which python)
/Users/fzzfbyx/pythonvirtual/tff/bin/python: Mach-O 64-bit executable x86_64

is that a problem?

**Comment by tackoo** [comment]
I am having the same issue. For some reason the script in the homepage doesn't seem to be installing the arm version on my M1 Mac.

**Comment by Willian-Zhang** [comment]
try `deactivate` and `conda deactivate` , also be sure to use terminal instead of any IDE before running the installation script .

**Comment by noahc1510** [comment]
I have the same issue, too. But I installed it on anaconda base on intel platform.

**Comment by Tioz90** [comment]
Same issue here in Apple Silicon Mac

**Comment by SevenChords** [comment]
same issue here, also using an Apple Silicon Mac, installed using anaconda3.8 since the new macs with the M1 chip ship with python version 3.9 and tensorflow_macos requires python 3.8

**Comment by mostsignificantbit** [comment]
> try `deactivate` and `conda deactivate` , also be sure to use Terminal.app instead of any IDE before running the installation script .

as well the initiation command should be triggered from an expected cpu process: i.e. using any x86 IDE built in terminal to start python will start the x86 python version of the universal app installed /usr/bin and lead to the unwanted results that are experienced here.

So what to do on a M1 Mac?
- don't install any additional python versions which are not universal, i.e. just install the one that comes from Apple with Xcode command line tools
- the correct one is at /usr/bin/python3 and using the command >$file /usr/bin/python3< should reveal both architectures
- use the macOS terminal.app (it's an arm64 process) to install the Tensorflow package (this will install the arm version)
- use the macOS terminal.app (not a x86 IDE / it's built in terminal, like VStudioCode or PyCharm) to start your virtual environment and run the scripts

**Comment by mostsignificantbit** [comment]
> same issue here, also using an Apple Silicon Mac, installed using anaconda3.8 since the new macs with the M1 chip ship with python version 3.9 and tensorflow_macos requires python 3.8

Mine came with 3.8.2 (I guess by installing the Xcode command line tools) . So I guess you installed something additionally that installed a non universal Python. You can check it with >file $(which python3)<

**Comment by fzzfbyx** [comment]
thank you so much.just delete all other python version and use the original python 3.8 which is in user/bin to create the virtualenv.and don't use any IDE.Problem solved.

**Comment by SevenChords** [comment]
can't do @fzzfbyx since all Apple Silicon macs ship with python 3.9 and don't have python 3.8 installed, you have to install tensorflow through another installation of python, and currently anaconda is the easiest to install on Apple Silicon macs

**Comment by fzzfbyx** [comment]
the verison of python in usr/bin is 3.8 and it has two architectures(the file name is python3).I just uninstall all other python downloaded from web and use that one.It worked.

**Comment by mostsignificantbit** [comment]
> can't do @mostsignificantbit since all Apple Silicon macs ship with python 3.9 and don't have python 3.8 installed, you have to install tensorflow through another installation of python, if you try the one installes in usr/bin you will get an error because that one is version 3.9 and tensorflow_macos needs python 3.8

please check again, mine shipped with 3.8.2 as pointed out above.
type in the terminal 

`file $(which python3)`

You should get a result like this, otherwise you overwrote your install:
```
/usr/bin/python3 (for architecture x86_64):	Mach-O 64-bit executable x86_64
/usr/bin/python3 (for architecture arm64e):	Mach-O 64-bit executable arm64e
```

`python3 --version`
`Python 3.8.2`

**Comment by SevenChords** [comment]
ok, just checked, when i first got it it shipped with 3.9, but i had to do a reinstall of macos because of some other thing, and now it has 3.8.2 installed, anyway, now it should work

**Comment by diogoribeiro09** [comment]
Sorry to bring back this topic from the dead but somehow i overwrote the default python path. When i do `file $(which python3)` I get:

`/Library/Frameworks/Python.framework/Versions/3.8/bin/python3: Mach-O 64-bit executable x86_64`

Is there a way to point it back to /usr/bin/python3 ?
